package.json

```diff
@@ -49,7 +50,7 @@
     "pako": "^1.0.10",
     "punycode": "^2.1.1",
     "trezor-connect": "^8.0.13",
-    "unchained-bitcoin": "0.0.7"
+    "unchained-bitcoin": "file:../unchained-bitcoin"
```

Comment:
Need to switch this to the correct `unchained-bitcoin` version.

Reply:
Done!